build.gradle

```diff
@@ -24,8 +26,12 @@ buildscript {
 
         common_utils_version = System.getProperty("common_utils.version", '2.9.0.0-SNAPSHOT')
         kafka_version  = '3.5.1'
-        jaxb_version = '2.3.8'
+        apache_cxf_version = '4.0.2'
+        open_saml_version = '4.3.0'
```

Comment:
Could we we exclude so far OpenSAML 4.3.0 version?

Reply:
+1, OpenSAML has a known issue with permissions that needs to be resolved before backport to 2.x.

There's a PR that will resolve the permissions issue here: https://github.com/opensearch-project/security/pull/2991

Reply:
Thanks for catching this